backend/small_eod/institution/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from generic.models import TimestampUserLogModel
+# from teryt_tree.models import JednostkaAdministracyjna
```

Comment:
Dlaczego?

Reply:
teryt_tree nie dziala z django3.0 - jest to spowodowane usunieciem pewnych funkcjonalnosci z django.utils

Reply:
Korzystamy z django-teryt-tree w innych projektach Stowarzyszenia. Czego brakuje? Czy jest utworzone w właściwym projekcie issues wskazujące na problem? Czy masz ochotę zrobić PR do tamtego projektu, aby to rozwiązać? Tam masz rozwiązane importowanie jednostek podziału terytorialnego, a skądś ponad 3k rekordów wziąć będziemy musieli.

Reply:
Pierwszy blad, ktory sie pojawia to:
ImportError: cannot import name 'python_2_unicode_compatible' from 'django.utils.encoding'

Jak rozumiem, ten dekorator mial zapewnic wsteczna kompatybilnosc z pythonem2. Moglbym sprobowac to naprawic, ale nie wiem jak wersjonowac ta paczke w pip, zeby dla starszych aplikacji dalej mozna bylo korzystac z tej biblioteki.

Nie tworzylem zadnego nigdzie Issue. Zaraz to zrobie.